# 前、中、后推杆的当前长度
float32 cur_length_f
float32 cur_length_b
float32 cur_length_m